pkg/render/guardian.go

```diff
@@ -277,6 +243,8 @@ func (c *GuardianComponent) container() []corev1.Container {
 				{Name: "GUARDIAN_PORT", Value: "9443"},
 				{Name: "GUARDIAN_LOGLEVEL", Value: "INFO"},
 				{Name: "GUARDIAN_VOLTRON_URL", Value: c.cfg.URL},
+				{Name: "VOLTRON_PACKET_CAPTURE_CA_BUNDLE_PATH", Value: c.cfg.TrustedCertBundle.MountPath()},
+				{Name: "VOLTRON_PROMETHEUS_CA_BUNDLE_PATH", Value: c.cfg.TrustedCertBundle.MountPath()},
```

Comment:
It seems weird that we're adding the same bundle twice, might be nice to modify guardian so we don't have two env variable to reference the same thing.

Reply:
That's the plan :+1: